Makefile.pre.in

```diff
@@ -1730,13 +1730,17 @@ tags::
 	ctags -w $(srcdir)/Include/*.h $(srcdir)/Include/cpython/*.h $(srcdir)/Include/internal/*.h
 	for i in $(SRCDIRS); do ctags -f tags -w -a $(srcdir)/$$i/*.[ch]; done
 	ctags -f tags -w -a $(srcdir)/Modules/_ctypes/*.[ch]
+	find $(srcdir)/Lib -type f -name "*.py" -not -name "test_*.py" -not -path "*/test/*" | ctags -f tags -w -a -L -
```

Comment:
This still will pick up some stuff that is not likely wanted, for example, the contents of `Lib/lib2to3/tests`. Also, some packages have the test folder inside like `Lib/unittest/test/`.

Reply:
It will exclude them in https://github.com/python/cpython/pull/17721/commits/4681826932ab9100fc20629b4908023a11139950